### ISSUE TITLE
Settings menu dark mode grey "background" is not actually the background

### ISSUE THREAD

**Issue by mozzius** [bluesky-social/social-app#1456]
**Describe the bug**

This probably bothers me more than it should, but they settings page in dark mode looks horrible for a couple reasons

1. The background gray is way too light - doesn't match anything else, and typically the "deeper" elements like the background should be darker than the things on top, like the menu options
2. It's just a View within the scrollview, not the screen's actual background color. This means when you overscroll on iOS you see the true black background which is very jarring

This issue is specifically for the second issue - it's be great if the grey color is on the scrollview contentContainer instead of within the scrollview

**To Reproduce**

1. Turn on dark mode
2. Look at settings screen
3. Scroll up (on iOS)

**Expected behavior**

Grey background should not move

**Screenshots**

![image](https://github.com/bluesky-social/social-app/assets/10959775/94ad9dbd-6bf7-4df5-a38a-cef5e166d07a)

**Details**

 - Platform:  iOS
 - Platform version: 17 beta

**Comment by pfrazee** [comment]
Yeah we can prob hit this when we do a design system rework. You're right that it's inverted in the wrong way.
